src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -290,6 +290,7 @@ public function stream($responses, float $timeout = null): ResponseStreamInterfa
             throw new \TypeError(sprintf('%s() expects parameter 1 to be an iterable of CurlResponse objects, %s given.', __METHOD__, \is_object($responses) ? \get_class($responses) : \gettype($responses)));
         }
 
+        $active = null; // see https://github.com/symfony/symfony/issues/32833
```

Comment:
please remove the comment, we don't link back to github, there's git blame for that